content/en/docs/installing-on-kubernetes/hosted-kubernetes/install-on-gke.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Deploy KubeSphere on GKE"
+keywords: 'Kubernetes, KubeSphere, GKE, Installation'
+description: 'How to install KubeSphere on GKE'
+
+weight: 2265
+---
+
+![KubeSphere+GKE](https://pek3b.qingstor.com/kubesphere-docs/png/20191123145223.png)
+
+This guide walks you through the steps of deploying KubeSphere on [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/).
+
+## Prepare a GKE Cluster
+
+- A standard Kubernetes in GKE is a prerequisite of installing KubeSphere. Go to the navigation menu and refer to the image below to create a cluster.
+
+![create-cluster-gke](https://ap3.qingstor.com/kubesphere-website/docs/create-cluster-gke.jpg)
+
+- In **Cluster basics**, select a Master version. The static version `1.15.12-gke.2` is used here as an example.
+
+![](https://ap3.qingstor.com/kubesphere-website/docs/master-version.png)
+
+- In **default-pool** under **Node Pools**, define 3 nodes in this cluster.
+
+![node-number](https://ap3.qingstor.com/kubesphere-website/docs/node-number.png)
+
+- Go to **Nodes**, select the image type and set the Machine Configuration as below. When you finish, click **Create**.
+
+![machine-config](https://ap3.qingstor.com/kubesphere-website/docs/machine-configuration.jpg)
+
+{{< notice note >}} 
+
+- Supported Kubernetes versions for KubeSphere 3.0.0: 1.15.x, 1.16.x, 1.17.x, 1.18.x.
+- Ubuntu is used for the operating system here as an example. For more information on supported systems, see Overview.
+- 3 nodes are included in this example. You can add more nodes based on your own needs especially in a production environment.
+- The machine type e2-medium (2 vCPU, 4GB memory) is for minimal installation. If you want to enable pluggable components or use the cluster for production, please select a machine type with more resources.
+- For other settings, you can change them as well based on your own needs or use the default value.
+
+{{</ notice >}} 
+
+- When the GKE cluster is ready, you can connect to the cluster with Cloud Shell.
+
+
+![cloud-shell-gke](https://ap3.qingstor.com/kubesphere-website/docs/cloud-shell.png)
+
+## Install KubeSphere
```

Comment:
Install KubeSphere => Install KubeSphere on GKE. This title will be easier to be included in the search engine.